src/Application/UI/Presenter.php

```diff
@@ -21,6 +21,7 @@
  * @property-read string $action
  * @property      string $view
  * @property      string $layout
+ * @property-read ITemplate|\stdClass $template
```

Comment:
But in UI\Control is these declaration: @property-read ITemplate $template

But PhpStorm cannot detect that $template type is \stdClass